src/pages/Spreadsheet.tsx

```diff
@@ -63,7 +86,11 @@ export const Spreadsheet = (): React.ReactElement => {
         same thing as the {calculator}.
       </p>
 
-      <p>{spreadsheet('Use the spreadsheet here')}</p>
+      <p>
+        <strong>{spreadsheet('Use the spreadsheet here →')}</strong>
```

Comment:
I personally find the arrow weird; in my mind it should be pointing to something, like 

use the spreadsheet here -> <link>

I'd prefer it went away here and in the subscribe form, but won't block on it.